netmanager/src/views/components/DataDisplay/DeviceView/DeviceDeployStatus.js

```diff
@@ -720,17 +738,16 @@ export default function DeviceDeployStatus({ deviceData, siteOptions }) {
               placement="top"
               disableFocusListener={runReport.successfulTestRun && !deviceData.isActive}
               disableHoverListener={runReport.successfulTestRun && !deviceData.isActive}
-              disableTouchListener={runReport.successfulTestRun && !deviceData.isActive}>
+              disableTouchListener={runReport.successfulTestRun && !deviceData.isActive}
+            >
               <span>
                 <Button
                   variant="contained"
                   color="primary"
-                  disabled={weightedBool(
-                    deployLoading,
-                    deviceData.isActive || !runReport.successfulTestRun || inputErrors
-                  )}
+                  disabled={weightedBool(deployLoading, deviceData.isActive || inputErrors)}
```

Comment:
Just removed the device test check on the deploy button

Reply:
Brilliant! :)